index.html

```diff
@@ -0,0 +1,109 @@
+        <ul>
+          <li>coding</li>
+          <li>programming</li>
+        </ul>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+          culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio
+          enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+          recusandae eius vitae.</p>
+
+      </section>
+      <br>
+      <section class="main-section" id="prerequisite">
+        <header>prerequisite</header>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+          culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio <code>tempore sequi</code>
+          enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+          recusandae eius vitae.</p>
+          <ul>
+            <li>basic GIT</li>
+            <li> VScode studio </li>
+          </ul>
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+            culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio
+            enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+            recusandae eius vitae.</p>
+          
+    </section>
+    <br>
+    <section class="main-section" id="html">
+        <header>html</header>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+          culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur <code>tempore sequi</code> sint distinctio
+          enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+          recusandae eius vitae.</p>
+          <ul>
+            <li>tags</li>
+            <li>script</li>
+          </ul>
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+            culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio
+            enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+            recusandae eius vitae.</p>
+          
+    </section>
+    <br>
+    <section class="main-section" id="CSS">
+        <header>CSS</header>
```

Comment:
use `h2` tag instead of `header`

Reply:
* Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
* Kindly explain why you request a change - it might be obvious for you but not for someone else

Reply:
please use the h2 tag instead of header, just remember from HTML and CSS good practices h2 are used to highlight important pieces of information and head up subsections on your page